docs/src/getting_started/toolsinstallation.md

```diff
@@ -21,15 +23,17 @@ alone can almost be considered a proper IDE. It should appear the moment you ent
 julia> 
 ```
 
-We won't go into every feature that the REPL provides, however here is a quick list that most Julia users should
-know to make the most of this shell:
+Not every feature that the REPL provides will be shown, however here is a quick
+list that most Julia users should know to make the most of this shell:
 
-- There are 5 Julia REPL modes that serve separate purposes: 
+* There are 5 Julia REPL modes that serve separate purposes: 
 
   1. The Julian (default) mode wherein one can write, load, and test out code
   2. Help mode for printing documentation for functions, keywords, etc.
-  3. Pkg mode for working with the included package manager and manipulating dependencies/packages
-  4. Shell mode to execute system commands (run a terminal from Julia which itself is running from a terminal!) 
+  3. Pkg mode for working with the included package manager and manipulating
+     dependencies/packages
+  4. Shell mode to execute system commands (run a terminal from Julia which
+     itself is running from a terminal!) 
   5. Search modes (forward, reverse) to look through your history of inputs
 
 |        Prompt         |         Key binding         |
```

Comment:
I wonder if it's worth mentioning how to get out of these modes, the fact that Pkg / Shell are sticky, etc